src/codegen.cpp

```diff
@@ -1522,6 +1529,125 @@ static const auto pointer_from_objref_func = new JuliaFunction<>{
             Attributes(C, {Attribute::NonNull}),
             None); },
 };
+
+// TODO: Better way to declare vectorized intrinsics.
```

Comment:
Usually we give the functions names, so that you can define these with less duplication as:
```c++
static const auto gc_loaded_v2_func = new JuliaFunction<>{
    "julia.gc_loaded.v2",
    [](LLVMContext &C) { return gc_loaded_vN_func(C, 2); }
    gc_loaded_vN_attrs
};
```